From ba036f669ff244cebd3068b9dd61f6cb20c61aba Mon Sep 17 00:00:00 2001
From: Paul Gortmaker <paul.gortmaker@windriver.com>
Date: Fri, 22 Feb 2013 14:32:42 -0500
Subject: [PATCH 033/859] lttng: fix reference to obsolete RT Kconfig variable.

The preempt-rt patches no longer use CONFIG_PREEMPT_RT in
the 3.4 (and newer) versions.  So even though LTSI doesn't
include RT, having this define present can lead to an easy
to overlook bug for anyone who does try to layer RT onto
the LTSI baseline.

Update it to use the currently used define name by RT.

Reported-by: Jim Somerville <Jim.Somerville@windriver.com>
Signed-off-by: Paul Gortmaker <paul.gortmaker@windriver.com>
---
 drivers/staging/lttng/lttng-events.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/staging/lttng/lttng-events.c b/drivers/staging/lttng/lttng-events.c
index 954e9ce..9dcd609 100644
--- a/drivers/staging/lttng/lttng-events.c
+++ b/drivers/staging/lttng/lttng-events.c
@@ -51,7 +51,7 @@ int _lttng_session_metadata_statedump(struct lttng_session *session);
 void synchronize_trace(void)
 {
 	synchronize_sched();
-#ifdef CONFIG_PREEMPT_RT
+#ifdef CONFIG_PREEMPT_RT_FULL
 	synchronize_rcu();
 #endif
 }
-- 
1.8.1.4

